packages/compiler/src/server/tspconfig/completion.ts

```diff
@@ -40,30 +85,31 @@ export async function provideTspconfigCompletionItems(
         nodePath[0] === "emit" &&
         targetType === "arr-item")
     ) {
-      const emitters = await emitterProvider.listEmitters(tspConfigFile);
+      const emitters = await emitterProvider.listLibraries(tspConfigFile);
       const items: CompletionItem[] = [];
       for (const [name, pkg] of Object.entries(emitters)) {
         if (!siblings.includes(name)) {
-          const item: CompletionItem = {
-            label: name,
-            kind: CompletionItemKind.Field,
-            documentation: (await pkg.getPackageJsonData())?.description ?? `Emitter from ${name}`,
-            insertText: `"${name}"`,
-          };
-          items.push(item);
+          const item = createCompletionItemWithQuote(
+            name,
+            (await pkg.getPackageJsonData())?.description ?? `Emitter from ${name}`,
+            tspConfigPosition,
+            target,
+          );
+          if (item) {
```

Comment:
if we can't create an item with quote, we should fall back to the original way instead of not adding the item